### ISSUE TITLE
Update Virtualbox Guest Addition from 5.0.2 to 5.0.4

### ISSUE THREAD

**Issue by joelhandwell** [chef/bento#448]
I'm using bento box of Ubuntu 14 with following
* https://github.com/test-kitchen/kitchen-vagrantt with vagrant-vbguest
* https://github.com/test-kitchen/kitchen-vagrant

Virtualbox 5.0.4 released on September 8th 2015, and as I upgrade, vagrant-vbguest started to update vbguest every time kitchen test runs and consuming longer time to test.

I can get around this by changing to [another box with 5.0.4 pre-installed](atlas.hashicorp.com/joelhandwell/boxes/ubuntu_trusty64_vbguest) but it would be good to have the 5.0.4 installed by default in Ubuntu 14 (and other boxes) 

I'm looking for the code which is in charge of VirtualBox Guest Addition version to make PR, but not sure if anyone else if already started to work on upgrading. 

Following is corresponding log I'm getting (you'll notice they do compilation, download, install etc taking long time):

```
GuestAdditions versions on your host (5.0.4) and guest (5.0.2) do not match.
       stdin: is not a tty
       Reading package lists...
       Building dependency tree...

       The following extra packages will be installed:
         cpp fakeroot gcc libfakeroot linux-headers-3.19.0-25 patch
       Suggested packages:
         cpp-doc dpkg-dev debhelper gcc-multilib manpages-dev autoconf automake1.9
         libtool flex bison gdb gcc-doc diffutils-doc
       The following NEW packages will be installed:
         cpp dkms fakeroot gcc libfakeroot linux-headers-3.19.0-25
         linux-headers-3.19.0-25-generic patch
       0 upgraded, 8 newly installed, 0 to remove and 0 not upgraded.
       Need to get 10.3 MB of archives.
       After this operation, 81.6 MB of additional disk space will be used.
       Get:1 http://us.archive.ubuntu.com/ubuntu/ trusty/main cpp amd64 4:4.8.2-1ubuntu6 [27.5 kB]
       Get:2 http://us.archive.ubuntu.com/ubuntu/ trusty/main gcc amd64 4:4.8.2-1ubuntu6 [5,098 B]
       Get:3 http://us.archive.ubuntu.com/ubuntu/ trusty-updates/main patch amd64 2.7.1-4ubuntu2.3 [86.4 kB]
       Err http://us.archive.ubuntu.com/ubuntu/ trusty-updates/main dkms all 2.2.0.3-1.1ubuntu5.14.04.1
         404  Not Found [IP: 91.189.91.24 80]
       Get:4 http://us.archive.ubuntu.com/ubuntu/ trusty/main libfakeroot amd64 1.20-3ubuntu2 [25.4 kB]
       Get:5 http://us.archive.ubuntu.com/ubuntu/ trusty/main fakeroot amd64 1.20-3ubuntu2 [55.0 kB]
       Get:6 http://us.archive.ubuntu.com/ubuntu/ trusty-updates/main linux-headers-3.19.0-25 all 3.19.0-25.26~14.04.1 [9,291 kB]
       Get:7 http://us.archive.ubuntu.com/ubuntu/ trusty-updates/main linux-headers-3.19.0-25-generic amd64 3.19.0-25.26~14.04.1 [742 kB]
       Fetched 10.2 MB in 2s (4,158 kB/s)
       Failed to fetch http://us.archive.ubuntu.com/ubuntu/pool/main/d/dkms/dkms_2.2.0.3-1.1ubuntu5.14.04.1_all.deb  404  Not Found [IP: 91.189.91.24 80]

       E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
       stdin: is not a tty
       Ign http://security.ubuntu.com trusty-security InRelease
       Ign http://us.archive.ubuntu.com trusty InRelease
       Get:1 http://security.ubuntu.com trusty-security Release.gpg [933 B]
       Ign http://us.archive.ubuntu.com trusty-updates InRelease
       Get:2 http://security.ubuntu.com trusty-security Release [63.5 kB]
       Ign http://us.archive.ubuntu.com trusty-backports InRelease
       Hit http://us.archive.ubuntu.com trusty Release.gpg
       Get:3 http://us.archive.ubuntu.com trusty-updates Release.gpg [933 B]
       Hit http://us.archive.ubuntu.com trusty-backports Release.gpg
       Hit http://us.archive.ubuntu.com trusty Release
       Get:4 http://us.archive.ubuntu.com trusty-updates Release [63.5 kB]
       Hit http://us.archive.ubuntu.com trusty-backports Release
       Get:5 http://security.ubuntu.com trusty-security/main Sources [94.1 kB]
       Hit http://us.archive.ubuntu.com trusty/main Sources
       Hit http://us.archive.ubuntu.com trusty/restricted Sources
       Hit http://us.archive.ubuntu.com trusty/universe Sources
       Get:6 http://security.ubuntu.com trusty-security/restricted Sources [2,061 B]
       Hit http://us.archive.ubuntu.com trusty/multiverse Sources
       Get:7 http://security.ubuntu.com trusty-security/universe Sources [30.5 kB]
       Hit http://us.archive.ubuntu.com trusty/main amd64 Packages
       Get:8 http://security.ubuntu.com trusty-security/multiverse Sources [2,330 B]
       Hit http://us.archive.ubuntu.com trusty/restricted amd64 Packages
       Get:9 http://security.ubuntu.com trusty-security/main amd64 Packages [343 kB]
       Hit http://us.archive.ubuntu.com trusty/universe amd64 Packages
       Hit http://us.archive.ubuntu.com trusty/multiverse amd64 Packages
       Hit http://us.archive.ubuntu.com trusty/main i386 Packages
       Hit http://us.archive.ubuntu.com trusty/restricted i386 Packages
       Hit http://us.archive.ubuntu.com trusty/universe i386 Packages
       Hit http://us.archive.ubuntu.com trusty/multiverse i386 Packages
       Hit http://us.archive.ubuntu.com trusty/main Translation-en
       Get:10 http://security.ubuntu.com trusty-security/restricted amd64 Packages [8,875 B]
       Hit http://us.archive.ubuntu.com trusty/multiverse Translation-en
       Get:11 http://security.ubuntu.com trusty-security/universe amd64 Packages [116 kB]
       Hit http://us.archive.ubuntu.com trusty/restricted Translation-en
       Hit http://us.archive.ubuntu.com trusty/universe Translation-en
       Get:12 http://us.archive.ubuntu.com trusty-updates/main Sources [234 kB]
       Get:13 http://security.ubuntu.com trusty-security/multiverse amd64 Packages [3,686 B]
       Get:14 http://security.ubuntu.com trusty-security/main i386 Packages [327 kB]
       Get:15 http://us.archive.ubuntu.com trusty-updates/restricted Sources [4,725 B]
       Get:16 http://us.archive.ubuntu.com trusty-updates/universe Sources [135 kB]
       Get:17 http://us.archive.ubuntu.com trusty-updates/multiverse Sources [5,143 B]
       Get:18 http://us.archive.ubuntu.com trusty-updates/main amd64 Packages [619 kB]
       Get:19 http://security.ubuntu.com trusty-security/restricted i386 Packages [8,846 B]
       Get:20 http://security.ubuntu.com trusty-security/universe i386 Packages [116 kB]
       Get:21 http://security.ubuntu.com trusty-security/multiverse i386 Packages [3,830 B]
       Get:22 http://security.ubuntu.com trusty-security/main Translation-en [187 kB]
       Get:23 http://us.archive.ubuntu.com trusty-updates/restricted amd64 Packages [15.4 kB]
       Hit http://security.ubuntu.com trusty-security/multiverse Translation-en
       Hit http://security.ubuntu.com trusty-security/restricted Translation-en
       Get:24 http://us.archive.ubuntu.com trusty-updates/universe amd64 Packages [312 kB]
       Get:25 http://security.ubuntu.com trusty-security/universe Translation-en [67.8 kB]
       Get:26 http://us.archive.ubuntu.com trusty-updates/multiverse amd64 Packages [11.9 kB]
       Get:27 http://us.archive.ubuntu.com trusty-updates/main i386 Packages [601 kB]
       Get:28 http://us.archive.ubuntu.com trusty-updates/restricted i386 Packages [15.1 kB]
       Get:29 http://us.archive.ubuntu.com trusty-updates/universe i386 Packages [313 kB]
       Get:30 http://us.archive.ubuntu.com trusty-updates/multiverse i386 Packages [12.1 kB]
       Get:31 http://us.archive.ubuntu.com trusty-updates/main Translation-en [300 kB]
       Hit http://us.archive.ubuntu.com trusty-updates/multiverse Translation-en
       Hit http://us.archive.ubuntu.com trusty-updates/restricted Translation-en
       Get:32 http://us.archive.ubuntu.com trusty-updates/universe Translation-en [166 kB]
       Hit http://us.archive.ubuntu.com trusty-backports/main Sources
       Hit http://us.archive.ubuntu.com trusty-backports/restricted Sources
       Hit http://us.archive.ubuntu.com trusty-backports/universe Sources
       Hit http://us.archive.ubuntu.com trusty-backports/multiverse Sources
       Hit http://us.archive.ubuntu.com trusty-backports/main amd64 Packages
       Hit http://us.archive.ubuntu.com trusty-backports/restricted amd64 Packages
       Hit http://us.archive.ubuntu.com trusty-backports/universe amd64 Packages
       Hit http://us.archive.ubuntu.com trusty-backports/multiverse amd64 Packages
       Hit http://us.archive.ubuntu.com trusty-backports/main i386 Packages
       Hit http://us.archive.ubuntu.com trusty-backports/restricted i386 Packages
       Hit http://us.archive.ubuntu.com trusty-backports/universe i386 Packages
       Hit http://us.archive.ubuntu.com trusty-backports/multiverse i386 Packages
       Hit http://us.archive.ubuntu.com trusty-backports/main Translation-en
       Hit http://us.archive.ubuntu.com trusty-backports/multiverse Translation-en
       Hit http://us.archive.ubuntu.com trusty-backports/restricted Translation-en
       Hit http://us.archive.ubuntu.com trusty-backports/universe Translation-en
       Ign http://us.archive.ubuntu.com trusty/main Translation-en_US
       Ign http://us.archive.ubuntu.com trusty/multiverse Translation-en_US
       Ign http://us.archive.ubuntu.com trusty/restricted Translation-en_US
       Ign http://us.archive.ubuntu.com trusty/universe Translation-en_US
       Fetched 4,184 kB in 7s (558 kB/s)

       stdin: is not a tty
       Reading package lists...

       Reading state information...
       The following extra packages will be installed:
         cpp fakeroot gcc libfakeroot linux-headers-3.19.0-25 patch
       Suggested packages:
         cpp-doc dpkg-dev debhelper gcc-multilib manpages-dev autoconf automake1.9
         libtool flex bison gdb gcc-doc diffutils-doc
       The following NEW packages will be installed:
         cpp dkms fakeroot gcc libfakeroot linux-headers-3.19.0-25
         linux-headers-3.19.0-25-generic patch
       0 upgraded, 8 newly installed, 0 to remove and 14 not upgraded.
       Need to get 65.4 kB/10.3 MB of archives.
       After this operation, 81.6 MB of additional disk space will be used.
       Get:1 http://us.archive.ubuntu.com/ubuntu/ trusty-updates/main dkms all 2.2.0.3-1.1ubuntu5.14.04.2 [65.4 kB]
       dpkg-preconfigure: unable to re-open stdin: No such file or directory
       Fetched 65.4 kB in 0s (937 kB/s)
       Selecting previously unselected package cpp.
       (Reading database ... 32984 files and directories currently installed.)
       Preparing to unpack .../cpp_4%3a4.8.2-1ubuntu6_amd64.deb ...
       Unpacking cpp (4:4.8.2-1ubuntu6) ...
       Selecting previously unselected package gcc.
       Preparing to unpack .../gcc_4%3a4.8.2-1ubuntu6_amd64.deb ...
       Unpacking gcc (4:4.8.2-1ubuntu6) ...
       Selecting previously unselected package patch.
       Preparing to unpack .../patch_2.7.1-4ubuntu2.3_amd64.deb ...
       Unpacking patch (2.7.1-4ubuntu2.3) ...
       Selecting previously unselected package dkms.
       Preparing to unpack .../dkms_2.2.0.3-1.1ubuntu5.14.04.2_all.deb ...
       Unpacking dkms (2.2.0.3-1.1ubuntu5.14.04.2) ...
       Selecting previously unselected package libfakeroot:amd64.
       Preparing to unpack .../libfakeroot_1.20-3ubuntu2_amd64.deb ...
       Unpacking libfakeroot:amd64 (1.20-3ubuntu2) ...
       Selecting previously unselected package fakeroot.
       Preparing to unpack .../fakeroot_1.20-3ubuntu2_amd64.deb ...
       Unpacking fakeroot (1.20-3ubuntu2) ...
       Selecting previously unselected package linux-headers-3.19.0-25.
       Preparing to unpack .../linux-headers-3.19.0-25_3.19.0-25.26~14.04.1_all.deb ...
       Unpacking linux-headers-3.19.0-25 (3.19.0-25.26~14.04.1) ...
       Selecting previously unselected package linux-headers-3.19.0-25-generic.
       Preparing to unpack .../linux-headers-3.19.0-25-generic_3.19.0-25.26~14.04.1_amd64.deb ...
       Unpacking linux-headers-3.19.0-25-generic (3.19.0-25.26~14.04.1) ...
       Processing triggers for man-db (2.6.7.1-1ubuntu1) ...
       Setting up cpp (4:4.8.2-1ubuntu6) ...
       Setting up gcc (4:4.8.2-1ubuntu6) ...
       Setting up patch (2.7.1-4ubuntu2.3) ...
       Setting up dkms (2.2.0.3-1.1ubuntu5.14.04.2) ...
       Setting up libfakeroot:amd64 (1.20-3ubuntu2) ...
       Setting up fakeroot (1.20-3ubuntu2) ...
       update-alternatives: using /usr/bin/fakeroot-sysv to provide /usr/bin/fakeroot (fakeroot) in auto mode
       Setting up linux-headers-3.19.0-25 (3.19.0-25.26~14.04.1) ...
       Setting up linux-headers-3.19.0-25-generic (3.19.0-25.26~14.04.1) ...
       Examining /etc/kernel/header_postinst.d.
       run-parts: executing /etc/kernel/header_postinst.d/dkms 3.19.0-25-generic /boot/vmlinuz-3.19.0-25-generic
       Copy iso file C:\Program Files/Oracle/VirtualBox/VBoxGuestAdditions.iso into the box /tmp/VBoxGuestAdditions.iso
       stdin: is not a tty
       mount: block device /tmp/VBoxGuestAdditions.iso is write-protected, mounting read-only
       Installing Virtualbox Guest Additions 5.0.4 - guest version is 5.0.2
       stdin: is not a tty
       Verifying archive integrity... All good.
       ............
       VirtualBox Guest Additions installer
       Removing installed version 5.0.2 of VirtualBox Guest Additions...
       Stopping VirtualBox Additions ...fail!
       (Cannot unload module vboxguest)
       Removing existing VirtualBox DKMS kernel modules ...done.
        ...done.
       Stopping VirtualBox Guest Addition service  ...done.
       Copying additional installer modules ...
       Installing additional modules ...
       Removing existing VirtualBox DKMS kernel modules ...done.
        ...done.
        ...done.
        ...done.
       You should restart your guest to make sure the new modules are actually used

       Installing the Window System drivers
       Could not find the X.Org or XFree86 Window System, skipping.
       An error occurred during installation of VirtualBox Guest Additions 5.0.4. Some functionality may not work as intended.
       In most cases it is OK that the "Window System drivers" installation failed.
```

**Comment by joelhandwell** [comment]
Hello @cheeseplus is there anything I can do for upgrading virtualbox version for the box in terms of making PR?

**Comment by cheeseplus** [comment]
@joelhandwell there isn't anything to do in terms of a PR - this requires updating our build farm and pushing a new set of boxes. This is something that is already in progress. 

This specific issue, at least the failure portion, is related to the use of the vagrant-vbguest plugin.

**Comment by joelhandwell** [comment]
@cheeseplus Thank you for the info. I'll wait for the new vbox version to be available.

**Comment by cheeseplus** [comment]
Fixed (updated to 5.0.6) efeb14a810551b06d7c5718639034cd9d3b45219

**Comment by joelhandwell** [comment]
Thanks!
